src/tests/ftest/launch.py

```diff
@@ -1059,6 +1219,10 @@ def main():
         "-r", "--rename",
         action="store_true",
         help="rename the avocado test logs directory to include the test name")
+    parser.add_argument(
```

Comment:
If we are archiving log files and configuration files from remote hosts would it make sense to include core files as part of that process?  I guess I'm on the fence as to if this should be a separate option or just included as part of `--archive`.

Reply:
We're not archiving the core files.  We are processing them with `gdb` to get a stack trace and then throwing the core file away.

Reply:
That's fair.  I'm good with the separate argument.